pkg/machines/components/vmDisksTab.jsx

```diff
@@ -106,6 +107,17 @@ const VmDisksTab = ({ idPrefix, disks, actions, renderCapacity, notificationText
                     }
 
                     columns.push(disk.diskSourceCell);
+
+                    if (provider === 'LibvirtDBus') {
+                        const removeDiskAction = RemoveDiskAction({
+                            dispatch,
+                            vm,
+                            target: disk.target,
+                            idPrefixRow,
+                        });
+                        columns.push(<div>{removeDiskAction}</div>);
+                    }
```

Comment:
out of interest: is that not possible to do with virsh, or do we just not care any more?

Reply:
We don't care anymore. At least what I got from previous discussions is that we stop feature development in virsh provider.